independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/project/CodestartResourceLoadersBuilder.java

```diff
@@ -146,10 +150,11 @@ private static List<ResourceLoader> getCodestartResourceLoaders(String baseCodes
         final List<ResourceLoader> codestartResourceLoaders = new ArrayList<>(codestartsArtifacts.size());
         for (Artifact a : codestartsArtifacts.values()) {
             try {
-                final File artifactFile = mvn.resolve(a).getArtifact().getFile();
+                final File artifactFile = mavenArtifactResolver.resolve(a).getArtifact().getFile();
                 codestartResourceLoaders.add(resolveFileResourceLoader(artifactFile));
             } catch (Exception e) {
-                throw new RuntimeException("Failed to resolve codestart artifact " + a, e);
+                log.warn("Unable to resolve codestart artifact for %s: %s", a, e.getMessage());
+                continue;
```

Comment:
This is the actual patch.

Reply:
Why the `continue`?

Reply:
It's not useful per se but you never know what might be added in the future so I wanted the intent to be extra clear.